src/skin/options.html

```diff
@@ -176,6 +176,15 @@ <h1><span class="i18n_options_title"></span></h1>
           <span class="i18n_options_dnt_policy_setting"></span>
         </label>
       </div>
+
+      <hr>
```

Comment:
I'm curious about the hr break here -- on one hand I'm a fan of breaking up chunks of information to make it more readable, on the other hand it's unclear to me how the two sections are thematically grouped.

Reply:
It separates flags that are enabled by default from flags that are disabled by default. This can probably be made clear by labeling the sections, but I just don't think this is important enough to bother with (any text changes involves translations ...), for now anyway.

Reply:
Thanks for pointing that out -- definitely makes sense then to separate them like that.